Spark/Dynamic_pricing/features/ride_duration_zscore.py

```diff
@@ -0,0 +1,21 @@
+from Spark.Dynamic_pricing.features.ride_durations import ride_durations
+from tecton.types import String, Timestamp, Float64, Field, Int64
+from tecton import on_demand_feature_view, RequestSource
+
+request_schema = [Field('origin_zipcode', String), Field('duration', Float64)]
+ride_request = RequestSource(schema=request_schema)
+output_schema = [Field('zscore_ride_duration', Float64)]
+
+@on_demand_feature_view(
```

Comment:
now that I think about it, z-score might not make the most sense here - I'd imagine that in dynamic pricing, the exact distance/duration of the ride is more relevant than the relative duration

maybe I should change this to an ODFV that just directly computes the duration/distance of the ride?